resources.py

```diff
@@ -6,9 +6,17 @@ class Message(object):
 	"""
 	Message templates
 	"""
-
-	NO_DEVICE_FOUND="There is no device with {dev_att}: {value}! Please check the device {dev_att}!"
-	NO_APPLICATION_FOUND="There is no application with {app_att}: {value}! Please check the application {app_att}!"
-	DEVICE_OFFLINE="The device is offline: {uuid}"
-	UNSUPPORTED_DEVICE="Unsupported device: {value}"
-	NO_KEY_FOUND="There is no key with {key_att}: {value}! Please check the key {key_att}!"
+	
+	# Exception Error Message
+	RESIN_NOT_LOGGED_IN = "You have to log in"
+	RESIN_REQUEST_ERROR = "Request error: {body}"
+	RESIN_KEY_NOT_FOUND = "Key not found: {key}"
+	RESIN_DEVICE_NOT_FOUND = "Device not found: {device}"
+	RESIN_APPLICATION_NOT_FOUND = "Application not found: {application}"
+	RESIN_MALFORMED_TOKEN = "Malformed token: {token}"
+	RESIN_INVALID_DEVICE_TYPE = "Invalid device type: {dev_type}"
+	RESIN_INVALID_OPTION = "Invalid option: {option}"
+	RESIN_MISSING_OPTION = "Missing option: {option}"
+	RESIN_NON_ALLOWED_OPTION = "Non allowed option: {option}"
+	RESIN_LOGIN_FAILED = "Invalid credentials"
+	RESIN_DEVICE_OFFLINE = "Device is offline: {uuid}"
```

Comment:
In both SDKs, we usually use Resin prefix with messages and errors so I want to keep this in term of consistency